### PR TITLE
Appveyor: Install DMD compiler

### PR DESCRIPTION
**Pull request by GoaLitiuM**

### CODE REVIEW

File: ci/install-dmd.ps1 [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+$ProgressPreference = "SilentlyContinue"
+
+# default installation directory
+$dmd_install = "C:\D"
+
+#echo "Fetching latest DMD version..."
+$dmd_version = [System.Text.Encoding]::ASCII.GetString((Invoke-WebRequest -URI "http://downloads.dlang.org/releases/LATEST").content)
+$dmd_url = "http://downloads.dlang.org/releases/2.x/$dmd_version/dmd.$dmd_version.windows.zip"
+$dmd_filename = [System.IO.Path]::GetFileName($dmd_url)
+$dmd_archive = Join-Path ($env:temp) $dmd_filename
```
Comment (nirbheek) [inline ci/install-dmd.ps1]: I think you probably want to save this under `C:\cache` and fetch a specific version instead of the latest one?

**Comment by nirbheek** [comment]
I cancelled the Travis build by hand because it seemed pointless to run it :)

File: ci/install-dmd.ps1 (hunk 1 above)
Comment (GoaLitiuM) [reply ci/install-dmd.ps1]: What's wrong with using the latest version? The script only stores the archive in system temp folder.

Comment (nirbheek) [reply ci/install-dmd.ps1]: Caching the output makes the CI more resilient to transient network failures (f.ex., if dlang.org is down). That's my main concern.

**Comment by GoaLitiuM** [comment]
This should be good to go. MSYS2 and Cygwin are skipped as I couldn't figure out how to fix linking errors in mixed C++ and D test, maybe someone with more knowledge on those platforms could step in and fix this, but I think the coverage for D Windows tests in MSVC environment should be sufficient for now.

**Pull request merged**